.github/workflows/common-release.yml

```diff
@@ -83,3 +83,19 @@ jobs:
             export GIT_COMMITTER_DATE="$(git log -1 --format=%aD ${{ inputs.versionPrefix }}${SEMVER_VERSION})"
             git push origin "${{ inputs.versionPrefix }}${SEMVER_VERSION}" -f
           fi
+
+      - name: Create Issue on Failed workflow
+        if: ${{ failure() }}
+        run: |
+          if curl -sL --fail\
+              -X POST \
+              -H "Accept: application/vnd.github+json" \
+              -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              https://api.github.com/repos/${{ github.repository }}/issues \
+              -d '{"title":"Release Pipeline failed","body":"For details see: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}","labels":["bug"]}' \
+              -o /dev/null; then
+              echo "Issue successfully created"
+          else
+              echo "Issue creation failed"
```

Comment:
do we want to `exit 1` here?

Reply:
I had that at first, but I figured it wasn't necessary because this part of the action only runs if the pipeline has failed.

Reply:
ok fair enough